web/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/MiniApp/page/MiniAppForm/index.tsx

```diff
@@ -333,9 +333,9 @@ export const MiniAppForm = (props: LinksFormProps) => {
               }
               items={countries}
               label=""
-              disabled={!isEditable || !isEnoughPermissions || !appMode}
+              disabled={!isEditable || !isEnoughPermissions}
               errors={errors.supported_countries}
-              required={appMode}
+              required
```

Comment:
@andy-t-wang Should this input be then on some other step?

Reply:
Yeah we are reorganizing everything with design updates but I'm waiting on them. Figured not to move anything until that